src/policy/policy.h

```diff
@@ -47,7 +45,7 @@ static const unsigned int MAX_STANDARD_P2WSH_SCRIPT_SIZE = 3600;
  * standard and should be done with care and ideally rarely. It makes sense to
  * only increase the dust limit after prior releases were already not creating
  * outputs below the new threshold */
-static const unsigned int DUST_RELAY_TX_FEE = 3000;
+static const unsigned int DUST_RELAY_TX_FEE = 1000000;
```

Comment:
Do we use `DUST_RELAY_TX_FEE`?

Reply:
Yes, in multiple place actually, when we compute the newly created tx

Reply:
What is it for exactly?

Reply:
And why is the value 1000000?

Reply:
Will work on dust in another PR. 
https://en.bitcoinwiki.org/wiki/Cryptocurrency_dust
And also remove bnb :)